CMakeLists.txt

```diff
@@ -4,6 +4,11 @@
 #   server and other components
 cmake_minimum_required(VERSION 3.21...3.22 FATAL_ERROR)
 
+# Suppress policy warnings due to normalized install directories
+if(POLICY CMP0177)
+  cmake_policy(SET CMP0177 OLD)
```

Comment:
Note that this is deprecated, so we should try and update our usage instead.

> The OLD behavior of a policy is [deprecated by definition](https://cmake.org/cmake/help/latest/manual/cmake-policies.7.html#manual:cmake-policies(7)) and may be removed in a future version of CMake.

Reply:
We will need to figure out a different way of how we normalize directories for the windows build. That is where the message comes from on the policy. I set the value to reduce a ton of messages at configure time.

Reply:
I didn't fully understand the docs, do you have an example?

Reply:
These lines cause the policy to complain. They only come up when compiling on MSYS2 or in Visual Studio. I honestly don't fully understand what the problem is, but cmake decided to change "normalized directories" behavior.

https://github.com/longturn/freeciv21/blob/master/CMakeLists.txt#L64-L68

here is the docs on the policy: https://cmake.org/cmake/help/latest/policy/CMP0177.html

I'll open an issue to see if we can correct how we are doing things.